Documentation/ApiOverview/Hooks/Configuration/Index.rst

```diff
@@ -93,9 +93,10 @@ extension keys. ::
   array and uses only the value (function reference)
 
 - **function\_reference :** A function reference using the syntax of
-  :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()`
-  or :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()`
-  depending on implementation of the hook.
+  :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()` as a function
+  or :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` as a class name
+  depending on implementation of the hook. A namespace function has the quoted string format :php:`'Foo\\Bar\\MyClassName->myUserFunction'`. A namespace class name can be in the quoted string format :php:`'Foo\\Bar\\MyClassName'` or in the unquoted form :php:`\Foo\Bar\MyClassName::class` (>= PHP 5.5). 
```

Comment:
Since this is the point of the PR you should mention that the FQCN must not start with a backslash.